### PR TITLE
update, unify cellranger module metadata

### PR DESCRIPTION
**Pull request by klkeys**
## PR checklist

Fixes missing `cellranger` module metadata from #2914. Slightly harmonizes the text between `cellranger` submodules.

### CODE REVIEW

File: modules/nf-core/cellranger/mkref/meta.yml [hunk 1]
```diff
@@ -15,15 +15,15 @@ tools:
 input:
   - fasta:
       type: file
-      description: fasta genome file
+      description: Reference genome FastA file
```
Comment (Emiller88) [inline modules/nf-core/cellranger/mkref/meta.yml:18]:
```suggestion
      description: Reference genome FASTA file
```

File: modules/nf-core/cellranger/mkfastq/meta.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name: cellranger_mkfastq
-description: Module to create fastqs needed by the 10x Genomics Cell Ranger tool. Uses the cellranger mkfastq command.
+description: Module to create FastQs needed by the 10x Genomics Cell Ranger tool. Uses the cellranger mkfastq command.
```
Comment (Emiller88) [inline modules/nf-core/cellranger/mkfastq/meta.yml:2]:
```suggestion
description: Module to create FASTQs needed by the 10x Genomics Cell Ranger tool. Uses the cellranger mkfastq command.
```

**Review by Emiller88 — CHANGES_REQUESTED** [review]
I think these are technically all uppercase (according to wikipedia) but I've seen it all sorts of ways, as long as we're consistent across everything.

**Review by klkeys — COMMENTED** [review]
changes from @Emiller88 incorporated 👍🏼

**Review by matthdsm — APPROVED** [review]

**Review by mashehu — APPROVED** [review]

**Pull request merged**